tracer/src/tracer.rs

```diff
@@ -105,11 +106,14 @@ struct TraceEvent {
 
 pub fn trace_point_log(event: &'static str) {
     let trace_event = TraceEvent {
+        // SAFETY: start is a variable that's local to the current thread
```

Comment:
`// SAFETY: start has been initialised as part of initialising the value of TRACER`